.github/workflows/test-coverage.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    branches: [ "develop" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
I like to always use this structure to keep standards steps. This is not an obligation, just a suggestion!
```
      - name: Checkout
        uses: actions/checkout@v3
```

The ideia is to have a name for each steps, to make it clear what the steps is doing when the job runs.